worker/pyproject.toml

```diff
@@ -0,0 +1,227 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "zimfarm_worker"
+version = "0.1.0"
+authors = [
+  { name = "Kiwix", email = "dev@kiwix.org" },
+]
+keywords = ["zimfarm"]
+requires-python = ">=3.13,<3.14"
+description = "zimfarm Worker"
+license = {text = "GPL-3.0-or-later"}
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.13",
+    "License :: OSI Approved :: GNU General Public License v3 or later (GPLv3+)",
+]
+dependencies = [
+    "paramiko == 3.5.1",
+    "PyJWT == 2.10.1",
+    "humanfriendly == 10.0",
+    "cryptography == 45.0.3",
+    "docker == 7.1.0",
+]
+
+[project.optional-dependencies]
+scripts = [
+    "invoke == 2.2.0",
+]
+lint = [
+    "black == 25.1.0",
+    "ruff == 0.11.10",
+]
+check = [
+    "pyright == 1.1.400",
+    "types-humanfriendly == 10.0.0"
+]
+
+test = [
+    "coverage == 7.8.0",
+    "pytest == 8.3.5",
+]
+
+dev = [
+    "pre-commit == 4.2.0",
+    "ipykernel == 6.29.5",
+    "zimfarm_worker[scripts]",
+    "zimfarm_worker[lint]",
+    "zimfarm_worker[test]",
+    "zimfarm_worker[check]",
+]
+
+
+[project.urls]
+Homepage = "https://github.com/openzim/zimfarm/worker"
+Donate = "https://www.kiwix.org/en/support-us/"
+
+[tool.hatch.build]
+exclude = [
+  "/.github",
+]
+
+[tool.hatch.build.targets.wheel]
+packages = ["src/task", "src/common", "src/manager"]
+
+[tool.hatch.build.targets.task]
+packages = ["src/task", "src/common"]
+
+[tool.hatch.build.targets.manager]
+packages = ["src/manager", "src/common"]
+
+[tool.hatch.metadata]
+allow-direct-references = true
+
+[tool.hatch.envs.default]
+features = ["dev"]
+
+[tool.hatch.envs.manager]
+dependences = [
+    "requests == 2.32.3",
+]
+
+[tool.hatch.envs.task]
+dependences = [
```

Comment:
Typo

Reply:
+ why do we need `SQLAlchemy` and don't need `requests`? Kinda weird for the task manager (but maybe this is for next PR ^^)